docs/redenomination.md

```diff
@@ -0,0 +1,123 @@
+---
+id: redenomination
+title: Redenomination of DOT
+sidebar_label: Redenomination of DOT
+---
+
+**The DOT redenomination is taking place on 21st August, now to be known as Denomination Day, at
```

Comment:
It might be easier for new users to understand renomination if there was a definition and a reason why it took place.

Reply:
Something like this would make it easier for people that are trying to integrate in this industry to understand :)

Reply:
I rewrote the first couple sentences, hopefully making an easier on-ramp for new readers.